cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3336,6 +3336,38 @@ TEST(TestArrowReadWrite, NonUniqueDictionaryValues) {
   }
 }
 
+TEST(TestArrowReadWrite, DictionaryIndexBidWidthsArePreservedOnRoundTrip) {
```

Comment:
Suggestion: 1) fix typo 2) make test name slightly shorter :)
```suggestion
TEST(TestArrowReadWrite, DictionaryIndexBitwidthRoundtrip) {
```